site/components/layout/Layout.tsx

```diff
@@ -0,0 +1,73 @@
+import Head from "next/head";
+import React, { PropsWithChildren, ReactElement } from "react";
+import Help from "../Help";
+import { ErrorInfo } from "../../interfaces";
+import { buildTitle } from "../../utils";
+import Footer from "./Footer";
+import PhaseBanner from "./PhaseBanner";
+
+interface LayoutProps {
+    title: string;
+    description: string;
+    errors?: ErrorInfo[];
+    hideCookieBanner?: boolean;
+    hideHelp?: boolean;
+}
+
+export const BaseLayout = ({
+    title,
+    description,
+    errors = [],
+    children,
+    hideHelp,
+}: PropsWithChildren<LayoutProps>): ReactElement => {
+    return (
+        <>
+            <Head>
+                <link rel="icon" href="/favicon.ico" />
+                <title>{buildTitle(errors, title || "Create Fares Data")}</title>
```

Comment:
Replace any instances of Create Fares Data with Create Transport Disruption Data

Reply:
done